subprojects/packagefiles/abseil-cpp/meson.build

```diff
@@ -13,14 +13,26 @@ cpp = meson.get_compiler('cpp')
 flags = cpp.get_supported_arguments('/DNOMINMAX', '-Wno-sign-compare', '-Wno-gcc-compat')
 add_project_arguments(flags, language: 'cpp')
 
+arch_cpp_flags = []
 hw_cpp_flags = []
 if host_machine.cpu_family() == 'x86_64'
   hw_cpp_flags += ['-maes', '-msse4.1']
 elif host_machine.cpu_family() == 'aarch64' and cpp.sizeof('void*') == 8
   hw_cpp_flags += ['-march=armv8-a+crypto']
 elif host_machine.cpu_family() == 'arm' and cpp.sizeof('void*') == 4
   hw_cpp_flags += ['-mfpu=neon']
+elif host_machine.cpu_family() == 'ppc' or host_machine.cpu_family() == 'ppc64'
```

Comment:
`elif host_machine.cpu_family().startswith('ppc')` or `elif host_machine.cpu_family() in ['ppc', 'ppc64']`
For future reference.